Light Vox Engine/Engine.cpp

```diff
@@ -253,11 +253,21 @@ void Engine::OnResize( uint32_t width, uint32_t height )
 void Engine::UsingInputFunc()
 {
     DEBUG_PRINT( "FPS: %f \n", 1.0f / time->GetDeltaFloatTime() );
-    printf( "FPS: %f \n", 1.0f / time->GetDeltaFloatTime() );
 }
 
 inline void Engine::Update()
 {
+    float dtfloat = time->GetDeltaFloatTime();
+    const static float speed = 2.f;
+    if ( inputManager->IsKeyDown( 0x41 ) ) // A
```

Comment:
Can't we use `VK_kEYCODE` for a bit more readable code?

Reply:
there's no VK_KEYCODE for awsd -- i can re-map it to the arrow keys for now though